### PR TITLE
[Outlook] (signature sample) Update requirement set and supported platform

### LINKED ISSUE #293: DefaultMinVersion="1.10" for outlook-set-signature
**Describe the bug**

[outlook-set-signature] sample use the signature API which is available from version 1.10, so manifest should reflect that.

**To Reproduce**

The Add-ins isn't compatible with version prior to 1.10

**Expected behavior**

Add-ins shouldn't be load if version 1.10 is not available

**Environment**
Example : Outlook 2016 shouldn't load the Add-ins

**Additional context**

https://docs.microsoft.com/en-us/javascript/api/requirement-sets/outlook/requirement-set-1.10/outlook-requirement-set-1.10?view=common-js-preview

### PR DESCRIPTION
**Pull request by samantharamon**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?             | yes |
| New feature?    | no |
| New sample?    | no |
| Related issues? | fixes #293 |

## What's in this Pull Request?

Updated the outlook-set-signature sample to–
- correct the minimum requirement set specified in the manifest to 1.10; and
- add support for Outlook on Mac new UI (preview).

## Guidance

Updated the outlook-set-signature sample to–
- correct the minimum requirement set specified in the manifest to 1.10; and
- add support for Outlook on Mac new UI (preview).

### CODE REVIEW

File: Samples/outlook-set-signature/README.md [hunk 1]
```diff
@@ -61,16 +62,16 @@ In this scenario, the add-in helps the user manage their email signature, even w
 
 ## Run the sample
 
-You can run this sample in Outlook on Windows or in a browser. The add-in web files are served from this repo on GitHub.
+You can run this sample in Outlook on Windows, Outlook on Mac (new UI in preview), or in a browser. The add-in web files are served from this repo on GitHub.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/outlook-set-signature/README.md:65]:
```suggestion
You can run this sample in Outlook on Windows, on Mac (new UI in preview), or in a browser. The add-in web files are served from this repo on GitHub.
```

File: Samples/outlook-set-signature/README.md [hunk 2]
```diff
@@ -111,14 +112,14 @@ If you prefer to host the web server for the sample on your computer, follow the
...
+1. Sideload `manifest-localhost.xml` in Outlook on the web, on Windows, or on Mac by following the manual instructions in the article [Sideload Outlook add-ins for testing](https://docs.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing).
 1. [Try out the sample!](#try-it-out)
 
 ## Key parts of this sample
 
 ### Configure event-based activation in the manifest
 
-The manifest configures a runtime that is loaded specifically to handle event-based activation. The following `<Runtime>` element specifies an HTML page resource id that loads the runtime on Outlook on the web. The `<Override>` element specifies the JavaScript file instead, to load the runtime for Outlook on Windows. Outlook on Windows doesn't use the HTML page to load the runtime.
+The manifest configures a runtime that is loaded specifically to handle event-based activation. The following `<Runtime>` element specifies an HTML page resource ID that loads the runtime on Outlook on the web and Outlook on Mac. The `<Override>` element specifies the JavaScript file instead, to load the runtime for Outlook on Windows. Outlook on Windows doesn't use the HTML page to load the runtime.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/outlook-set-signature/README.md:122]:
```suggestion
The manifest configures a runtime that is loaded specifically to handle event-based activation. The following `<Runtime>` element specifies an HTML page resource ID that loads the runtime on Outlook on the web and on Mac. The `<Override>` element specifies the JavaScript file instead, to load the runtime for Outlook on Windows. Outlook on Windows doesn't use the HTML page to load the runtime.
```

File: Samples/outlook-set-signature/README.md [hunk 3]
```diff
@@ -139,11 +140,11 @@ The add-in handles two events that are mapped to the `checkSignature()` function
 
 ### Handling the events and using the setSignatureAsync API
 
-When the user creates a new message or appointment, Outlook will load the files specified in the manifest to handle the `OnNewMessageCompose` and `OnNewAppointmentOrganizer` events. Outlook on the web will load the `autorunweb.html` page, which then also loads `autorunweb.js` and `autorunshared.js`.
+When the user creates a new message or appointment, Outlook will load the files specified in the manifest to handle the `OnNewMessageCompose` and `OnNewAppointmentOrganizer` events. Outlook on the web and Outlook on Mac will load the `autorunweb.html` page, which then also loads `autorunweb.js` and `autorunshared.js`.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/outlook-set-signature/README.md:143]:
```suggestion
When the user creates a new message or appointment, Outlook will load the files specified in the manifest to handle the `OnNewMessageCompose` and `OnNewAppointmentOrganizer` events. Outlook on the web and on Mac will load the `autorunweb.html` page, which then also loads `autorunweb.js` and `autorunshared.js`.
```

File: Samples/outlook-set-signature/README.md [hunk 4]
```diff
@@ -139,11 +140,11 @@ The add-in handles two events that are mapped to the `checkSignature()` function
...
 The `autorunweb.js` file contains a version of the `insert_auto_signature` function used specifically when running on Outlook on the web. The [setSignatureAsync() API cannot be used in Outlook on the web for appointments](https://docs.microsoft.com/javascript/api/outlook/office.body?view=outlook-js-preview#setSignatureAsync_data__options__callback_). Therefore, `insert_auto_signature` inserts the signature into a new appointment by directly writing to the body text of the appointment.
 
-The `autorunshared.js` file contains the `checkSignature` function that handles the events from Outlook. It also contains additional code that is shared and loaded when the add-in is used in Outlook on the web and Outlook on Windows. On Outlook on Windows, this file is loaded directly and `autorunweb.html` and `autorunweb.js` are not loaded.
+The `autorunshared.js` file contains the `checkSignature` function that handles the events from Outlook. It also contains additional code that is shared and loaded when the add-in is used in Outlook on the web, Outlook on Windows, and Outlook on Mac. On Outlook on Windows, this file is loaded directly and `autorunweb.html` and `autorunweb.js` are not loaded.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/outlook-set-signature/README.md:147]:
```suggestion
The `autorunshared.js` file contains the `checkSignature` function that handles the events from Outlook. It also contains additional code that is shared and loaded when the add-in is used in Outlook on the web, on Windows, and on Mac. In Outlook on Windows, this file is loaded directly and `autorunweb.html` and `autorunweb.js` are not loaded.
```

**Review by davidchesnut — APPROVED** [review]
Looks good. Thanks!

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**